website/src/content/docs/docs/language-basics/directives.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Directives
+---
+
+Directives are predefined annotations that attach to the syntax nodes unlike decorators which will cary over with `model is`, `op is`, etc. This means any syntax node is able to have a directive(e.g `alias`).
+
+There is 2 available directives:
```

Comment:
```suggestion
There are 2 available directives:
```

Reply:
Actually - should we just say
```suggestion
These are the available directives:
```
so we don't have to remember to update that number whenever we add/remove one?